conda/deprecations.py

```diff
@@ -285,6 +335,20 @@ def _get_module(self, stack: int) -> tuple[ModuleType, str]:
 
```

Comment:
FYI the original `import inspect` is still on the line above

Reply:
Argh, forgot to add that to the commit!